extensions/fs/internal.m

```diff
@@ -1090,6 +1090,41 @@ static int fs_pathFromBookmark(lua_State *L) {
     return 1 ;
 }
 
+/// hs.fs.readFileContents(path) -> string | nil, string
+/// Function
+/// Reads the contents of a file at a given path.
+///
+/// Parameters:
+///  * path - The path to the file you want to read.
+///
+/// Returns:
+///  * A string containing the contents of the file or `nil` if an error occurs.
+///  * An error message if an error occurs.
+static int fs_readFileContents(lua_State *L) {
+    LuaSkin *skin = [LuaSkin shared] ;
+    [skin checkArgs:LS_TSTRING, LS_TBREAK] ;
+    
+    NSString *filePath = [skin toNSObjectAtIndex:1];
+    NSURL *url = [NSURL fileURLWithPath:filePath];
+    
+    NSError *error = nil;
+    NSStringEncoding encoding;
```

Comment:
What is the encoding variable used for?

Reply:
Nothing - it's just used to store the `NSStringEncoding` type when the `NSString` is initialised. I guess we could return it, if that's useful?